testutil/src/main/java/org/itsallcode/openfasttrace/testutil/importer/lightweightmarkup/AbstractLightWeightMarkupImporterTest.java

```diff
@@ -485,7 +501,33 @@ void testItemIdSupportsUTF8Characaters()
                         .title("Die Implementierung muss den Zustand einzelner Zellen ändern")
                         .description("Ermöglicht die Aktualisierung des Zustands von lebenden und toten Zellen"
                                 + " in jeder Generation.")
-                        .location("umlauts", 2).addNeedsArtifactType("arch")
+                        .location("umlauts", 2 + titleLocationOffset)
+                        .addNeedsArtifactType("arch")
                         .build()));
     }
+
+    @Test
+    void testHeaderBelongsToNextItem()
+    {
+        assertImport("file",
+                """
+                        ${title("Item 1", 1)}
+                        `req~item1~1
+                        Item 1 description
+
+                        ${title("Item 2", 1)}
+                        `req~item2~1
+                        Item 2 description
+                        """,
+                contains(item().id(SpecificationItemId.createId("req", "item1", 1))
+                        .title("Item 1")
+                        .description("Item 1 description")
+                        .location("file", 2 + titleLocationOffset)
+                        .build(),
+                        item().id(SpecificationItemId.createId("req", "item2", 1))
```

Comment:
:+1: Good test!

Indentation looks strange, though.

Reply:
The formatter did it ;) Is it better now?